demo/tests/perfs/test_perfs_rec.perf.yml

```diff
@@ -94,7 +52,33 @@ update-form-without-changes:
 - db: SELECT ... FROM "formidable_default" WHERE "formidable_default"."field_id" IN (...)
 - db: SELECT ... FROM "formidable_validation" WHERE "formidable_validation"."field_id" IN (...)
 - db: SELECT ... FROM "formidable_access" WHERE "formidable_access"."field_id" IN (...)
+get-context-form:
+- db: 'SELECT ... FROM "django_session" WHERE ("django_session"."expire_date" > # AND "django_session"."session_key" = #)'
+- db: 'SELECT ... FROM "formidable_formidable" WHERE "formidable_formidable"."id" = #'
+- db: 'SELECT ... FROM "formidable_field" WHERE "formidable_field"."form_id" = # ORDER BY "formidable_field"."order" ASC'
+- db: 'SELECT ... FROM "formidable_access" WHERE ("formidable_access"."access_id" = # AND NOT ("formidable_access"."level" = #) AND "formidable_access"."field_id" IN (...))'
+- db: SELECT ... FROM "formidable_item" WHERE "formidable_item"."field_id" IN (...) ORDER BY "formidable_item"."order" ASC
+- db: SELECT ... FROM "formidable_validation" WHERE "formidable_validation"."field_id" IN (...)
+- db: SELECT ... FROM "formidable_default" WHERE "formidable_default"."field_id" IN (...)
+retrieve-form:
+- db: 'SELECT ... FROM "django_session" WHERE ("django_session"."expire_date" > # AND "django_session"."session_key" = #)'
+- db: 'SELECT ... FROM "formidable_formidable" WHERE "formidable_formidable"."id" = #'
+- db: 'SELECT ... FROM "formidable_field" WHERE "formidable_field"."form_id" = # ORDER BY "formidable_field"."order" ASC'
+- db: SELECT ... FROM "formidable_item" WHERE "formidable_item"."field_id" IN (...) ORDER BY "formidable_item"."order" ASC
+- db: SELECT ... FROM "formidable_default" WHERE "formidable_default"."field_id" IN (...)
+- db: SELECT ... FROM "formidable_validation" WHERE "formidable_validation"."field_id" IN (...)
+- db: SELECT ... FROM "formidable_access" WHERE "formidable_access"."field_id" IN (...)
+update-form-with-changes:
```

Comment:
no more INSERT / UPDATE ?

Reply:
As I mentioned in the description of the PR:

> Please note that the .yml files diff are a bit confusing, because git can't figure out which lines belong to which tests. You may want to inspect the result files instead of the diff to ensure that now we're actually creating complete forms in our tests using the HTTP API.

If you want to compare the number of queries:

| test name | master | this branch |
|-|-|-|
| accesses-list | 0 | 0| 
| create-form | 12 | 52| 
| get-context-form | 3 | 7| 
| retrieve-form | 3 | 7 |
| update-form-with-changes | 24 | 2| 
| update-form-without-changes|  48 | 2| 
| validate-form | 3 | 7| 

I surely need to see why the ``update-form-with(out)-changes`` are like this, but for the create-form, the diff **was** confusing.

### How did I get to that?

```python
import yaml
data = yaml.load(open('test_perfs_rec.perf.yml'))
result = map(lambda item: (item[0], len(item[1])), data.items())
result = list(result)
print(result)
```

Reply:
ok, I know the issue. there was a 422 error at runtime. That's how you learn that you need to check the status code of your API call, even if you think that they're all fine. Patch on its way.

Reply:
updated the table above.